templates/centos6.tpl.yaml

```diff
@@ -66,7 +66,7 @@ objects:
       metadata:
         name: ${NAME}
       spec:
-        pvc:
+        storage:
           accessModes:
             - ReadWriteMany
```

Comment:
One principle benefit of using 'storage' is that you no longer need to specify the accessModes parameter.  CDI will look up the storage class and select the best access mode based on what provisioner is being used.  Would it be possible to remove accessModes from each of these templates?

Reply:
Sure I can remove it